### ISSUE TITLE
[Bug]: <Title>

### ISSUE THREAD

**Issue by joseortiz3** [timescale/timescaledb#6657]
### What type of bug is this?

Configuration, Unexpected error, Other

### What subsystems and features are affected?

Packaging, Platform/OS

### What happened?

`timescaledb-2.14.1.dll` requires the OpenSSL v1.1.1 libraries:
- `libcrypto-1_1-x64.dll` to be on system path
- `libssl-1_1-x64.dll` to be on system path

`timescaledb-2.13.0.dll` requires the OpenSSL v3.x libraries:
- `libcrypto-3-x64.dll` to be on system path
- `libssl-3-x64.dll` to be on system path

For some reason there was a change (regression) in the OpenSSL version required to run TimescaleDB. [This does not match what the documentation says.](https://docs.timescale.com/self-hosted/latest/install/installation-windows/)

**Suggested**: either 1) change the documentation or 2) change the required libraries.

Additionally: Change the documentation so user verifies OpenSSL version as step 1. Right now it is weirdly before step number 1 and I didn't even see it to be honest.

Also, maybe add a repo test requiring that a certain version of OpenSSL is used, if it might happen again?

Temporary Fix:

Users trying to upgrade from TimescaleDB version 2.13.0 to 2.14.1 (and maybe others) will encounter "module not found" error when PostgreSQL tries to use the library ``timescaledb-2.14.1.dll`. The missing modules in question are likely the OpenSSL 1.1.1 libraries mentioned before. You can install the correct version of OpenSSL1.1.1 using [a convenient installer from this website](https://slproweb.com/products/Win32OpenSSL.html) and make sure the required library is available from the command line.

Optionally you can install a dependency walker like [this more modern one](https://github.com/lucasg/Dependencies) and verify that your issues are missing these `.dll`.

### TimescaleDB version affected

2.14.1

### PostgreSQL version used

16.2

### What operating system did you use?

Windows 11 Pro 64

### What installation method did you use?

Other

### What platform did you run on?

On prem/Self-hosted

### How can we reproduce the bug?

```bash
Try to install 2.14.1 on a Windows x64 postgres without OpenSSL1.1.1 by following the docs
```

**Comment by brianewell** [comment]
Can confirm that manually installing OpenSSL v1.1.1 into a Windows 64 system fixes this.

**Comment by billy-the-fish** [comment]
Agreed solution, create a page in About Timescale called Third-party products that lists compatibilities between versions.

**Comment by mpw-eng** [comment]
Note that some Timescaledb users (like me) of are not developers and these bugs really slow us down.  This issue took me all day to figure out while trying to migrate from psql 14 to 16, and finding a Win10 installer for OpenSSL v1.1.1 was tricky too!  These two files must be in the Windows path environment variable for it to work;
- `libcrypto-1_1-x64.dll`
- `libssl-1_1-x64.dll`

Binaries found here: [https://wiki.overbyte.eu/arch/openssl-1.1.1w-win64.zip](url)

@joseortiz3 - Thank you for reporting this bug - I would never have solved this otherwise.

**Comment by adschmu** [comment]
@jnidzwetzki 

> Thanks for getting back. Since TimescaleDB 2.13.2 requires OpenSSL 1.1.1, you should also copy these DLLs (libcrypto-1_1-x64.dll and libssl-1_1-x64.dll) to the PostgreSQL directory. Could you try this and check if this solves the problem in your installation?

There is no 2.13.2 release and no statement I found that OpenSSL 1.x is used instead of 3.x again.

To address the obvious confusion, would you kindly state and/or describe what the current **intended** status for this subject is?

1. There is a decision to use OpenSSL 3 due to security concerns and the developers have the power to choose which version to use.
2. There is a decision like above, but you decided to go back to OpenSSL 1
3. You do not have the power to choose, and you rely on some other dependency for that. You now have to find out why this changed accidentally between 2.13.x and 2.14.x

Answering this question would already help to plan future deployments, even if the issue is not fixed yet.
